editor/src/components/canvas/controls/guideline-controls.tsx

```diff
@@ -26,6 +41,35 @@ export const GuidelineControls = React.memo(() => {
     )
   }, 'GuidelineControls strategyMovedSuccessfully')
 
+  const metadata = useEditorState((store) => store.editor.jsxMetadata, 'JSX metadata')
+
+  const { strategyIntendedBounds, snappingGuidelines } = useEditorState(
+    (store) => store.editor.canvas.controls,
+    'Strategy intended bounds and snapping guidelines',
+  )
+
+  const parentAndSiblings = collectParentsAndSiblings(
+    metadata,
+    strategyIntendedBounds.map((_) => _.target),
```

Comment:
Please don't use underscores here. We typically use those to denote a param that isn't used.

Reply:
fixed!